app/uk/gov/hmrc/trackingconsentfrontend/assets/src/config/config.ts

```diff
@@ -0,0 +1,21 @@
+export const getWelshLanguageEnabled = () => false
+
+const getPort = (): string | undefined => {
+    const scriptTag: HTMLScriptElement | null = document.querySelector('script[data-id="tracking-consent-frontend"]')
+    if (scriptTag === null) {
+        return undefined
+    }
+    const port: string | null = scriptTag.getAttribute('data-port')
```

Comment:
I'm nervous about reading the port from a data attribute which could accidentally or maliciously be incorrect.  We know the port already, or we can read it from the `src` attribute.

Reply:
Why? What threat do you have in mind? How would an attacker be able to maliciously alter it?

Reply:
Port now extracted from src of current script.